src/main/java/com/mycompany/app/application/StockQuoteApplication.java

```diff
@@ -0,0 +1,47 @@
+package com.mycompany.app.application;
+
+import java.util.*;
+import java.text.SimpleDateFormat;
+import com.mycompany.app.service.*;
+import com.mycompany.app.model.*;
+
+public class StockQuoteApplication {
+
+    /**
+     * A main method which enables program execution
+     *
+     * @param args an array that should contain as elements:
+     *       {@code String} representations of a symbol, start date, and end date
+     */
+    public static void main(String[] args) {
+
```

Comment:
there are a few things wrong here. 
1)         StockServiceFactory stockServiceFactory = new StockServiceFactory();
	
is incorrect the StockServiceFactory class should have a single static method and a private constructor so it can't be created via new.  You will only ever need the static methods in this class so creating an instance of it is not required. You enforce this by providing a private constructor. 

Next, the stockServiceFactory.createStockService method should not take an argument. Remember, we want to hide the implementation details (the the type of stock service ) from the results of the code. (I know some examples of Factories do it the way you have done it here, but that's incorrect).  I will discuss this again in this week's chat. I know last week's chat recording didn't work which is a shame.